fastlane/Fastfile

```diff
@@ -42,6 +42,9 @@ platform :ios do
       type: "adhoc",
       readonly: true
     )
+    increment_build_number(
+      build_number: ENV["CIRCLE_BUILD_NUM"]
```

Comment:
FYI: Not exactly scope of this ticket but this will improve the patch versioning in AppCenter making use of the CircleCI build count. I hope this will reduce confusion around finding the right builds.